### DISCUSSION TITLE
Course Audit

### DISCUSSION THREAD

**Discussion by vvukku** [snigi-gupta/AttendingUniversityAtBuffalo#5] [General]
Can someone please help me to understand the core idea behind the course audit and the process to enroll for the course audit?
I've already registered for 12 credits, is the course audit free of cost in this case?

Thank you in Advance,
Vaishnavi Vukku.

**Comment by snigi-gupta** [comment]
@vvukku ,

Course audit is a trial class sort of thing. Until the add/drop period you can attend as many classes as you want and 'audit' a course. Meaning, attend the classes, look at the logistics of the classes, manage your workload and see what suit you best for you. If you want you can change your course and also drop classes you don't like. This period is for you to decide what you will study for the rest of the semester. Once the add/drop deadline is passed, you will be charged for those many courses that you have finally enrolled. Or those many courses. 

Do note, that for that entire add/drop period professors conduct the classes as normal. So for example, if you do enroll in a course and you are given an assignment on the first day (which is due in 1 week) and you do plan to continue in that course then you will have to make sure you submit that assignment. Your scores, submissions, and everything related to the course are counted/or taken into account for your marks or points even during the audit week. 

I hope this helps you in understanding the audit week.
